dependency/pom.xml

```diff
@@ -44,8 +44,7 @@
 		<narayana.version>5.3.2.Final</narayana.version>
 		<cxf.version>3.1.6</cxf.version>
 		<logback.version>1.1.7</logback.version>
-		<spring-cloud-commons.version>1.2.0.RELEASE</spring-cloud-commons.version>
-		<spring-cloud-netflix.version>1.3.0.RELEASE</spring-cloud-netflix.version>
+		<spring-cloud-commons.version>1.1.0.RELEASE</spring-cloud-commons.version>
```

Comment:
we shall use a spring cloud parent to manage the dependencies, or we may have version conflicts